app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    class TargetsController < ApiController
+      def create
+        @target = TargetService.new(target_params, current_user, params[:latitude], params[:longitude]).create
```

Comment:
Why are not sending latitude and longitude on target_params?

Reply:
Due to them not being part of the Target table (i only store location, not the separate coordinates) it interferes with the creating process.
Should i add those attributes to the table?